packages/fiori/src/UploadCollection.js

```diff
@@ -73,7 +73,7 @@ const metadata = {
 
 		/**
 		 * By default there will be drag and drop overlay shown over the <code>ui5-upload-collection</code> when files
-		 * are dragged. If you don't intend to use drag and drop, set this property to <code>true</code>
+		 * are dragged. If you don't intend to use drag and drop, add this attribute.
```

Comment:
If you don't intend to use drag and drop, set this property.

Reply:
Done.